Pacman move frame

julia-jpg opened the pull request:
Pacman moved to branch dev. Main modified with the insertion of pacman in the gui

Commit:
adding image pacman in Main.java

Commit:
Add function isEatable in pacman.java and add PacmanFoodTest.java

Commit:
fixed and added pacman.png

Commit:
Added test PacmanFoodTest.java for pacman.java for eating

roht0515 commented on src/main/java/pacman/Main.java:
```diff
@@ -11,13 +14,21 @@
     public Main(){
         this.setLayout(null);
-        this.setSize(200,200);
-		this.setVisible(true);
```
Be careful with indentation

roht0515 commented on src/main/java/pacman/Main.java:
```diff
@@ -11,13 +14,21 @@
+        this.setSize(900,900);
+        p = new pacman(30,30,true);
 
-       // w = new ArrayList <wall>();     //create arraylist for outer wall
-        //fill outer wall with -
-        //Initialize object wall
```
Remove comment code if it doesn't work

roht0515 commented on src/main/java/pacman/Main.java:
```diff
@@ -1,7 +1,10 @@
+class Main extends JFrame{
 
     //attrib
     private pacman p;
```
Specify the name of pacman for another descriptive name

Commit:
Pacman moves with the arrows on the keyboard on a frame

Fixed code review observations at Main.java and added pacmanDraw.java

Commit:
First clean code in every document.java

Commit:
Second fast clean code of every document.java

Commit:
Changes of every tab to spaces in order to get it uniform

Commit:
Third clean code of every document.java

Commit:
fourth clean code of every document.java (removing unnecesary spaces)

Commit:
fixed code review errors and added comments to methods

julia-jpg committed:
files after pull

julia-jpg committed:
Pacman moves in the Jframe using Pacman.java methods,... was modified return methods Pacman.java because I needed int to move in Jframe ......added new Class ActionKeyPacman.java thats interacts between GUI and Pacman Logic

julia-jpg committed:
Modified test PacmanTest.java

julia-jpg committed:
pacman does not walk on the walls

julia-jpg committed:
added walls to the frame
added class DrawComponents.java because in this class draw pacman and walls for now

julia-jpg committed:
added all the walls to the frame

julia-jpg committed:
fixed test moveRightPacman and added files .class to gitignore
